app/src/pages/playground/ModelConfigButton.tsx

```diff
@@ -186,10 +186,21 @@ function AzureOpenAiModelConfigFormField({
     [updateModelConfig]
   );
 
+  const debouncedUpdateEndpoint = useMemo(
+    () =>
+      debounce((value: string) => {
+        updateModelConfig({
+          configKey: "endpoint",
+          value,
+        });
+      }, 250),
+    [updateModelConfig]
+  );
+
   return (
     <>
       <TextField
-        key={`${instance.id}-modelName-${instance.model.modelName}`}
+        key="model-name"
```

Comment:
I have no idea why these keys were like this. But it causes them to fully un-mount and lose focus